### ISSUE TITLE
Django admin: Review org model permissions for "analyst" role in the admin

### ISSUE THREAD

**Issue by Katherine-Osos** [cisagov/manage.get.gov#2896]
### Issue description

Currently, the permissions for all tables and fields related to the org model are the same for superusers and analysts. We need to review those elements to determine whether the analyst role needs to be modified.

### Acceptance criteria

- [ ] Review all the tables and fields related to org model
  - May want to set up a call with someone who can describe how all the tables and fields interact and the impact of adjusting permissions.
- [ ] Identify changes that need to be made to the permissions
- [ ] Create dev ticket to make those changes

### Links to other issues

Blocker: I suggest waiting until all elements of the org model have been implemented before doing this assessment (e.g. member management)

**Comment by h-m-f-t** [comment]
> Blocker: I suggest waiting until all elements of the org model have been implemented before doing this assessment (e.g. member management)

I agree with this, and so I don't think this should go in m3. I agree it's "analyst experience", but m3 seems like the early work around this topic...
